admin-manual/installation/installation.rst

```diff
@@ -386,27 +451,31 @@ and enabled before the archivematica install. It can be done with:
    sudo -u archivematica bash -c " \
    set -a -e -x
    source /etc/sysconfig/archivematica-storage-service
-   cd /usr/share/archivematica/storage-service
-   /usr/lib/python2.7/archivematica/storage-service/bin/python manage.py migrate
-   /usr/lib/python2.7/archivematica/storage-service/bin/python manage.py collectstatic --noinput
+   cd /usr/lib/archivematica/storage-service
+   /usr/share/python/archivematica-storage-service/bin/python manage.py migrate
    ";
 
-* And now, we enable and start the archivematica-storage-service and it’s nginx frontend
+* Now enable and start the archivematica-storage-service, rngd (needed for encrypted spaces) and the nginx frontend:
```

Comment:
Rngd?

Reply:
Googling rngd, I don't see any capitalized variants, so I'm gonna leave it lowercase.